src/test/java/uk/gov/hmcts/dts/fact/services/MapitServiceTest.java

```diff
@@ -124,4 +129,42 @@ void shouldReturnOptionalEmptyIfBlankPostcodeForPartial() {
         assertThat(result).isNotPresent();
         verifyNoInteractions(mapitClient);
     }
+
+    @Test
+    void localAuthorityExistsShouldReturnTrueForValidLocalAuthorityNames() {
+        LinkedHashMap<String, MapitArea> mockAreaData = new LinkedHashMap<>();
+        mockAreaData.put("100", new MapitArea("100", "Birmingham City Council", "MTD"));
```

Comment:
Java v9 provides a useful utility method to initialise a map with one line. You can try using Map.of() for this case.

Reply:
I've updated the code to use the Map.of() initializer here.